extensions/PhabBugz/lib/Util.pm

```diff
@@ -168,6 +168,16 @@ sub edit_revision_policy {
     return request('differential.revision.edit', $data);
 }
 
+sub add_comment_to_revision {
```

Comment:
This is not yet implemented but relates to:

https://trello.com/c/9edMlHrl/399-2-when-locking-down-a-revision-because-of-unknown-security-groups-i-want-to-see-a-helpful-message-in-the-revision-comments-and-t